transformer_lens/components.py

```diff
@@ -593,6 +598,33 @@ def forward(
             )
             / self.attn_scale
         )  # [batch, head_index, query_pos, key_pos]
+
```

Comment:
Might be worth adding a comment that the original paper calculates this once and applies to the mask
instead when training (which is more efficient)? Not sure about this one, because for our purposes
your approach seems to be the key one.